modes/lisp-mode/hyperspec.lisp

```diff
@@ -0,0 +1,1006 @@
+(in-package :lem-lisp-mode/internal)
```

Comment:
This might be a good idea to separate the packages. e.g. `lem-lisp-mode/hyperspec`